pkgs/development/tools/minizinc/ide.nix

```
@@ -1,26 +1,29 @@
-{ lib, mkDerivation, fetchFromGitHub, qtbase, qtwebengine, qtwebkit, qmake, minizinc }:
+{ lib, stdenv, fetchFromGitHub, qt6, minizinc, makeWrapper, Cocoa }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, qtbase, qtwebsockets, qmake, makeWrapper, Cocoa }:
```

Since `qt6Packages.callPackage` is used.